BLAS/SRC/zgemmt.f

```diff
@@ -0,0 +1,570 @@
+*> \brief \b ZGEMMT
+*
+*  =========== DOCUMENTATION ===========
+*
+* Online html documentation available at
+*            http://www.netlib.org/lapack/explore-html/
+*
+*  Definition:
+*  ===========
+*
+*       SUBROUTINE ZGEMMT(UPLO,TRANSA,TRANSB,N,K,ALPHA,A,LDA,B,LDB,BETA,
+*                         C,LDC)
+*
+*       .. Scalar Arguments ..
+*       COMPLEX*16 ALPHA,BETA
+*       INTEGER K,LDA,LDB,LDC,N
+*       CHARACTER TRANSA,TRANSB, UPLO
+*       ..
+*       .. Array Arguments ..
+*       COMPLEX*16 A(LDA,*),B(LDB,*),C(LDC,*)
+*       ..
+*
+*
+*> \par Purpose:
+*  =============
+*>
+*> \verbatim
+*>
+*> ZGEMMT  performs one of the matrix-matrix operations
+*>
+*>    C := alpha*op( A )*op( B ) + beta*C,
+*>
+*> where  op( X ) is one of
+*>
+*>    op( X ) = X   or   op( X ) = X**T,
+*>
+*> alpha and beta are scalars, and A, B and C are matrices, with op( A )
+*> an n by k matrix,  op( B )  a  k by n matrix and  C an n by n matrix.
+*> Thereby, the routine only accesses and updates the upper or lower
+*> triangular part of the result matrix C. This behaviour can be used,
+*> the resulting matrix C is known to be symmetric.
```

Comment:
I would add "if" and "Hermitian":
"This behaviour can be used **if** the resulting matrix C is known to be **Hermitian or** symmetric."

Reply:
I updated the corresponding parts in the documentation.